.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,7 @@
 - [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
 - [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
 - [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
+- [ ] Is your test running fine `brew test --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
```

Comment:
Is `--build-from-source` really a valid option for `brew test`?

Reply:
Yes, `--build-from-source `name doesn't sound right, while testing just replaced `build` with `test` it worked for me(sorry new up here 😅)
Just checked it, it works without this flag too and seems better like `brew audit`.

Will make the change and update to `brew test <formula>`